scss/components/_accordion.scss

```diff
@@ -43,4 +43,4 @@
       padding-top: 0;
     }
   }
-}
+}
```

Comment:
When modifying this component please also thoroughly check that card groups, card decks and card columns are all functioning and displaying correctly. We should definitely include those use-cases in our demo page.

Reply:
Can you explain further? I don't think accordions are meant to be combined with card groups/decks/column. I see all four of those as mutually exclusive options for laying out multiple cards.

Reply:
I don't think we will need to modify the accordion any further. And as for the other card layout types, those are all working fine on the demo page. And they will be given explicit examples in the upcoming Panels section that I am working on.

Reply:
I was referring to the card component